src/components/ServerDetails/ServerDetails.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Details } from '../Details/Details';
+import { Label } from '../Label/Label';
+import { IServerDetailsProps } from './ServerDetails.Props';
+import { Icon } from '../Icon/Icon';
+import './ServerDetails.scss';
+
+import { IServerCountersData } from './ServerDetails.Props';
+export class ServerDetails extends React.PureComponent<IServerDetailsProps, any> {
+
+    constructor(props?: IServerDetailsProps) {
+        super(props);
+    }
+
+    public render() {
+        const name = this._checkNameLen(this.props.serverName);
+
+        return (
+            <div className={classNames('server-details', this.props.serverStatusClass)}>
+                <div className={'server-details-header'}>
+                    <Label className="server-name" title={this.props.fqdmServerName}>{name}</Label>
+                    <Icon 
+                        className={classNames('disk-icon')} 
+                        iconName={'icon-LoadWithErrors'} 
+                        title={'Disks\n' + this._createTooltipText(this.props.disks)}/>
+                    { this.props.numberOfUsers && 
+                        <Icon data-users={this.props.numberOfUsers}
+                            iconName={'icon-User'}
+                            title={this.props.numberOfUsers + ' number of users online'}/>
+                    }
+                    {this.props.hasCloseButton && 
+                        <Icon disabled={ false }
+                            className={'dialog-button dialog-button-close'}
+                            onClick={this._dismiss.bind(this)} 
+                            iconName={'icon-Delete'}/>
+                    }
+                    <hr/>
+                </div>
+                {this.props.children}
+                <div className={'counters-container'}>
+                    {this._createCountersTiles(this.props.countersData)}
+                </div>
+            </div>
+        );
+    }
+
+    private _dismiss() {
+        this.props.onDismiss(this.props.serverId);
+    }
+
+    private _checkNameLen(name: string) : string {
+        if (name.length > 15) {
```

Comment:
can we change this logic to use css property text-overflow: ellipsis. Different font and font sizes may cause unexpected behavior here